controllers/rest/operation_initializers.go

```diff
@@ -225,6 +226,37 @@ func UserDefinedInitializer(ctxt context.Context, api *RESTAPI, path string, met
 		ctxt = context.WithValue(ctxt, weoscontext.EVENT_STORE, eventStore)
 	}
 
+	for _, resp := range operation.Responses {
```

Comment:
move this to the RouteInitializer because essentially api.e.File is setting up a route